containers/CompoundAndEarn/ListItem/index.js

```diff
@@ -9,24 +9,105 @@ import DetailItem from 'parts/Compound/CompoundListItem/DetailItem';
 import getUserBoost from 'utils/helpers/getUserBoost';
 import getProperAction from 'utils/helpers/getProperAction';
 import { isEmpty } from 'utils/helpers/utility';
+import { useCompoundAndEarnContract } from 'contexts/compound-and-earn-context';
+import { useWeb3React } from '@web3-react/core';
 
 const ListItem = ({
-  pool
+  pool,
+  modal,
+  setModal
 }) => {
   const { gauges, snowconeBalance, totalSnowcone } = useContracts();
+  const [timerRefresh, setTimerRefresh] = useState(undefined);
```

Comment:
when representing a missing value please use `null` instead of  `undefined`.

Reply:
solved, thanks